js/app.js

```diff
@@ -5,35 +5,48 @@ const books = document.querySelector('#books-list');
 const addBtn = document.getElementById('form');
 let booksArray = [];
 
-const data = localStorage.getItem('booksList');
-if (data === null) {
-  booksArray = [];
-} else {
-  booksArray = JSON.parse(data);
-}
+class Book {
+  constructor(title, author) {
+    const data = localStorage.getItem('booksList');
+    if (data === null) {
+      booksArray = [];
+    } else {
+      booksArray = JSON.parse(data);
+    }
+    this.title = title;
+    this.author = author;
+  }
 
-function displayBooks() {
-  let htmlCode = '';
-  booksArray.forEach((item, index) => {
-    htmlCode += `<div>
-    <h2>${item.title}</h2>
-    <p>${item.author}</p>
-    <button onclick='removeBook(${index})' id='remove-btn'>remove</button>
-    <hr>
-    </div>`;
-  });
-  books.innerHTML = htmlCode;
-}
+  static displayBooks() {
+    let htmlCode = '';
+    booksArray.forEach((item, index) => {
+      htmlCode += `<div>
+      <h2>${item.title}</h2>
+      <p>${item.author}</p>
+      <button onclick='Book.removeBook(${index})' id='remove-btn'>remove</button>
+      <hr>
+      </div>`;
+    });
+    books.innerHTML = htmlCode;
+  }
+
+  static addBook(title, author) {
+    booksArray.push({
```

Comment:
- [ ] [OPTIONAL] Try to use array methods that don't mutate the array, e.g. `concat`, `filter` and `slice` instead of `push`, `pop` and `splice`. if you're interested have a look at [this article](https://alistapart.com/article/why-mutation-can-be-scary/) in which it is very clearly explained why you should preferably use immutable array methods.